lib/errors.js

```diff
@@ -208,6 +208,14 @@ class ServiceInstanceNotFound extends NotFound {
 }
 exports.ServiceInstanceNotFound = ServiceInstanceNotFound;
 
+
+class SpaceOrOrgNotFound extends NotFound {
```

Comment:
where is this used? Looks like an unused error. Also even if required, should rather use the error `NotFound` with the required message. Having such a specific NotFound errors not required.